cds_migrator_kit/rdm/migration/stats/event_generator.py

```diff
@@ -11,6 +11,25 @@
 from datetime import datetime
 
 
+def flag_robots_and_COUNTER(entry):
+    """Return a tuple of booleans.
+
+    First element is the value of is_bot and the second the flag to mark the event as
+    `before_COUNTER` as we don't have information if the registered event was from a bot
+    or not.
+    """
+    # filter out robot events
+    is_bot = entry.get("bot")
+    before_COUNTER = False
+
+    if is_bot is None:
+        # We migrate them as valid and we mark them for future discoverability
+        before_COUNTER = True
+        is_bot = False
+
+    return is_bot, before_COUNTER
```

Comment:
Super minor: if you want to be even more explicit

```suggestion
    is_bot_missing = "bot" in entry
    if is_bot_missing:
        # when is_bot is missing, we assume that it was a human
        # but we mark this uncertainty with before_COUNTER =True
        is_bot = False
        before_COUNTER = True
    else:
       is_bot = entry["bot"]
       before_COUNTER = True

    return is_bot, before_COUNTER
```